src/Microsoft.ApplicationInsights.AspNetCore/Implementation/TelemetryConfigurationOptionsSetup.cs

```diff
@@ -81,6 +81,15 @@ public void Configure(TelemetryConfiguration configuration)
             {
                 module.Initialize(configuration);
             }
+
+            foreach (ITelemetryProcessor processor in configuration.TelemetryProcessors)
+            {
+                ITelemetryModule module = processor as ITelemetryModule;
```

Comment:
Ehm, from https://github.com/Microsoft/ApplicationInsights-dotnet/blob/develop/src/Core/Managed/Shared/Extensibility/ITelemetryModule.cs

```csharp
/// <summary>
    /// Represents an object that supports initialization from <see cref="TelemetryConfiguration"/>.
    /// </summary>
```
That is pretty much all there is to it :)